CRM/Export/BAO/ExportProcessor.php

```diff
@@ -639,6 +639,9 @@ public function setQueryFields($queryFields) {
     $queryFields['world_region']['context'] = 'country';
     $queryFields['state_province']['context'] = 'province';
     $queryFields['contact_id'] = ['title' => ts('Contact ID'), 'type' => CRM_Utils_Type::T_INT];
+    // Set the label to gender for gender_id as we it's ... magic (not in a good way).
+    // In other places the query object offers e.g contribution_status & contribution_status_id
+    $queryFields['gender_id']['title'] = ts('Gender');
```

Comment:
Why is this a thing?

Reply:
Does the comment not explain? Basically because it's all weird & scary on this field

Reply:
@seamuslee001 is this blocking merge - I did a fix on @mlutfy's custom data export issue but it's failing tests because it's all jumbled up with this PR